apps/examples/control_log/Kconfig

```diff
@@ -0,0 +1,16 @@
+
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_CTRLOG
+	bool "network logging tool"
```

Comment:
Is this a tool? Then how about moving this into apps/system, not example?

Reply:
Thanks for your suggestion. I'll move it to apps/system when it is stable and supports more network module later.